deploy-board/deploy_board/templates/deploys/deploy_details.tmpl

```diff
@@ -83,6 +83,22 @@
             {% include "builds/build_details.tmpl" %}
         </td>
     </tr>
+    <tr>
```

Comment:
Let's hide `account` column if `deploy_accounts` list is empty:
```
{% if deploy_accounts %}
        <td>Account</td>
        <td>
            {% for account in deploy_accounts %}
                {% if account.legacy_name %}
                <div class="deployToolTip btn btn-xs btn-default host-btn">
                {{ account.legacy_name }}
                </div>
                {% else %}
                <a class="deployToolTip btn btn-xs btn-default host-btn" href="/clouds/accounts/{{ account.cloudProvider }}/{{ account.cell }}/{{ account.id }}">
                    {{ account.data.ownerId }} / {{ account.name }}
                </a>
                {% endif %}
            {% endfor %}
        </td>
    </tr>
{% endif %}

```

Reply:
updated, thanks